### PR TITLE
feat: add ci to build release

### PR DESCRIPTION
**Pull request by miax-gevu**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - build: stable-x86
+          os: ubuntu-latest
+          rust: stable
+          target: x86_64-unknown-linux-gnu
+          qemu: i386
+#       - build: stable-aarch64
```
Comment (trusch) [inline .github/workflows/release.yaml:79]: Why is it commented out?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install packages (Ubuntu)
+      if: matrix.os == 'ubuntu-latest'
+      shell: bash
+      run: |-
+          sudo apt-get update
+          sudo apt-get install -y openssl ca-certificates pkg-config cmake libssl-dev
```
Comment (trusch) [inline .github/workflows/release.yaml:103]: Are you sure that those packages are not in the default image? They are kinda common and the ubuntu image that github uses is not just the docker.io/library/ubuntu image as it seems.

**Review by trusch — COMMENTED** [review]
This looks good in general, but I'd like to see this working with all the architectures :)

File: .github/workflows/release.yaml (hunk 1 above)
Comment (miax-gevu) [reply .github/workflows/release.yaml:79]: To enable it later if we want those to be able to compile

File: .github/workflows/release.yaml (hunk 2 above)
Comment (miax-gevu) [reply .github/workflows/release.yaml:103]: You are right, they might not be needed

**Review by trusch — APPROVED** [review]

**Pull request merged**